SteamKit2/SteamKit2/Steam/WebAPI/WebAPI.cs

```diff
@@ -210,6 +210,14 @@ public sealed class AsyncInterface : DynamicObject, IDisposable
                 RegexOptions.Compiled | RegexOptions.IgnoreCase
             );
 
+            /// <summary>
+            /// Gets or sets the URI which is used to execute WebAPI requests.
+            /// When overriden, "secure" option is ignored. This also affects internal SteamKit requests.
+            /// </summary>
+            /// <value>
+            /// Endpoint URI which is used to execute WebAPI requests. For example, "http://steam-api.local/proxy".
+            /// </value>
+            public static string HttpEndpoint { get; set; }
```

Comment:
How do you propose allowing to override the endpoint for calls that SK executes? (e.g. SteamDirectory)